### PR TITLE
Rs 2022 08 25

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 7/07_01_CIEL07_1C.txt [hunk 1]
```diff
@@ -133,24 +141,24 @@ Putting that aside.
...
 -- Page 84, Offset 42706. Choice.
 -- 食堂でカロリーの高い定食を頼もう。
- Go to the cafeteria and order something with a lot of calories.
+ Order something calorific at the cafeteria.
 }
 [sha:bc5e30b25d51412364b6ce7cb4c32d08388fef1d]{
 -- Page 84, Offset 42707. Choice.
 -- 購買でカレーパンを買い占めよう。
- Go to the school store and buy up a bunch of curry bread.
+ Buy up all the curry bread at the school store.
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_1C.txt:157]: 'all' makes this come across too strongly

**Review by Petrikow — COMMENTED** [review]
curry

File: script/Ciel/Day 7/07_01_CIEL07_1C.txt [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 -- Page 74, Offset 14979.
 -- 「オース、何日かぶりですかね遠野クン！
-"'Sup! Been a few days, Tohno-kun!
+"'Sup! Long time no see, Tohno-kun!
 }
 [sha:eb3d781be3db55cb608cf58b02540414ebbfb738]{
 -- Page 74, Offset 14980. Mods: @k, @e.
 -- おおかた季節の変わり目で風邪でも引いた……
-Guess you got down with some fever now that the seasons are changing...
+You catch a cold to go with the change in seasons or something...
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_1C.txt:21]: "change of seasons" sounds more natural and looks like ngram agrees

File: script/Ciel/Day 7/07_01_CIEL07_1C.txt [hunk 3]
```diff
@@ -72,56 +75,61 @@ Didn't you take some time off out of health concerns? Well, Ciel-senpai told me
...
- then I'm counting on you to introduce her to me, alright?"
+So if she's the type of girl I imagine――
+with long, black hair and a meek, frail personality that's practically begging to be protected――
+then I'm counting on you to introduce her to me, alright?"
 }
 [sha:6331c0e5e1cf4238e3ccc176d39188b3a9ebef98]{
 -- Page 79, Offset 14992.
 -- 「……なに。おまえ、そういうステレオタイプの年下、好きなの？」
-"What. Are you into cliche young girls like that?
+"What. Are you seriously into cliche young girls like that?"
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_1C.txt:90]: cliché
we're a french group after all

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 7/07_01_CIEL07_1C.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Ciel/Day 7/07_01_CIEL07_1C.txt:157]: I mean, he's a hungry lad. I think my main issue was with this formulation of 'buy up' - didn't sit right to me, feels like you have to buy up _on_ curry bread, or buy up _all_ the curry bread. I'll weaken it a bit.

**Pull request merged**